cli/tests/environment-install.bats

```diff
@@ -88,7 +88,25 @@ teardown() {
   "$FLOX_BIN" init
   run "$FLOX_BIN" install not-a-package
   assert_failure
-  assert_output --partial "failed to resolve \`not-a-package'"
+  assert_output --partial "could not install not-a-package"
+}
+
+@test "'flox install' provides suggestions when package not found" {
+  "$FLOX_BIN" init
+  run "$FLOX_BIN" install package
+  assert_failure
+  assert_output --partial "Here are a few other similar options:"
+  assert_output --partial "options with 'flox search package'"
+}
+
+@test "'flox install' provides curated suggestions when package not found" {
+  "$FLOX_BIN" init
+  run "$FLOX_BIN" install java
+  assert_failure
+  assert_output --partial "Try 'flox install jdk' instead."
+  assert_output --partial "Here are a few other similar options:"
+  assert_output --partial "$ flox install "
+  assert_output --partial "options with 'flox search jdk'"
```

Comment:
nonblocking suggestion: test the multiple packages case

Reply:
done